src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -1,17 +1,24 @@
 // Node modules.
 import React from 'react';
-import moment from 'moment';
 import download from 'downloadjs';
+import moment from 'moment';
 // Relative imports.
 import * as customPropTypes from '../prop-types';
 
-const onDownloadClick = url => event => {
-  // Escape early if we're not on IE.
-  if (!navigator.msSaveBlob) {
-    return;
+// Helper to check if the URL is the same as our current origin.
+const isURLSameOrigin = url => {
+  const matches = url.match(/^(https?:)?\/\/([^/]+)/);
+
+  // If URI is not absolute or protocol-relative then it is always same-origin.
+  if (!matches) {
+    return true;
   }
 
-  // Prevent browser default behavior.
+  return window.location.host === matches[2];
```

Comment:
We might be able to just match it like `url.startsWith(document.location.origin)`. I'm not sure, but I'm guessing the protocol might matter for the download too. They _should_ be https anyway.

Reply:
Good idea! I'll update that!